src/components/Heading/Heading.module.scss

```diff
@@ -7,6 +7,7 @@
   border-radius: $border-radius-small;
   min-width: 0;
   @include heading-padding();
+  margin: revert;
```

Comment:
why exactly is this here?

Reply:
heading component contains the default margin which received from the browser, removing it will be a breaking change

Reply:
so worth adding a comment and make sure the chromatic visual tests are ok

Reply:
Adding a comment and will check chromatic tests before merge